src/electionguard_gui/services/db_service.py

```diff
@@ -0,0 +1,18 @@
+from pymongo import MongoClient
+from pymongo.database import Database
+
+
+class DbService:
+    """Responsible for instantiating a database"""
+
+    _db_password: str
+
+    def __init__(self, db_password: str) -> None:
+        self._db_password = db_password
+
+    def get_db(self) -> Database:
+        client: MongoClient = MongoClient(
+            "localhost", 27017, username="root", password=self._db_password
```

Comment:
In a future version this will need to be changed to an environment variable so that the guardian devices can be setup to get the data from the central database.

Reply:
Good point, I'll add it now